actors/common/src/main/java/org/sunbird/learner/actors/PageManagementActor.java

```diff
@@ -506,18 +553,16 @@ private void createPage(Request actorMessage) {
 
   private void updatePageDataCacheHandler(Response response, Map<String, Object> pageMap) {
     // update DataCacheHandler page map with new page data
-    new Thread(
-            () -> {
-              if (JsonKey.SUCCESS.equalsIgnoreCase((String) response.get(JsonKey.RESPONSE))) {
-                String orgId = "NA";
-                if (pageMap.containsKey(JsonKey.ORGANISATION_ID)) {
-                  orgId = (String) pageMap.get(JsonKey.ORGANISATION_ID);
-                }
-                DataCacheHandler.getPageMap()
-                    .put(orgId + ":" + (String) pageMap.get(JsonKey.PAGE_NAME), pageMap);
-              }
-            })
-        .start();
+    new Thread(() -> {
+      if (JsonKey.SUCCESS.equalsIgnoreCase((String) response.get(JsonKey.RESPONSE))) {
+        String orgId = "NA";
+        if (pageMap.containsKey(JsonKey.ORGANISATION_ID)) {
+          orgId = (String) pageMap.get(JsonKey.ORGANISATION_ID);
+        }
+        cacheUtil.put(ActorOperations.GET_PAGE_DATA.getValue(), orgId + ":" + (String) pageMap.get(JsonKey.PAGE_NAME),
```

Comment:
Where is the hashing logic which we discussed?

Reply:
It is in project util. so it can be used in all the actors if needed